### ISSUE TITLE
suggestion: Way to disable notices

### ISSUE THREAD

**Issue by DjDeveloperr** [denodrivers/postgres#254]
There should be a way to disable these notices:

![img](https://cdn.discordapp.com/attachments/768920745618833478/811960577596850236/unknown.png)

Because not everyone wants it in their applications.

**Comment by andreespirela** [comment]
I agree. Logs should be totally optional, it is up to the developer to decide whether they want their `stdout` to be altered by other packages.

**Comment by al6x** [comment]
+1 it breaks the logs and looks ugly

**Comment by al6x** [comment]
If you need a quick fix

```TypeScript
// Hiding notice warnings
import { parseNotice } from "postgres/connection/warning.ts"
import { Connection } from "postgres/connection/connection.ts"

(Connection.prototype as something).processNotice = function(msg: something): something {
  return parseNotice(msg)
}
```

**Comment by imjamesb** [comment]
Any new quick fixes for disabling notices? Really annoying having them show up in my logs. The skipping notices are getting on my nerves to no end.
